src/main/webapp/WEB-INF/static/js/trialmanager/location/locationService.js

```diff
@@ -0,0 +1,94 @@
+(function () {
+	'use strict';
+
+	angular.module('manageTrialApp').factory('locationService', ['$http', '$q', 'studyContext', 'serviceUtilities',
```

Comment:
Just to note here that I've added a subtask in IBP-4284 to create angular test for this service